js/index.js

```diff
@@ -1,6 +1,37 @@
 // Iteration 1: Names and Input
-//
+
+let hacker1 = 'Jonathan';
+console.log(`The driver's name is ${hacker1}`);
+let hacker2 = 'Amanda';
+console.log(`The driver's name is ${hacker2}`);
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+} else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`);
+} else if (hacker1.length = hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
 
 // Iteration 3: Loops
+//3.1
+
+console.log(hacker1.toUpperCase());
```

Comment:
3.1 iteration asks you to also separate letter with space ;)
So result should look like this `"J O N A T H A N"`